demos/CMakeLists.txt

```diff
@@ -35,3 +35,6 @@ create_demo(Demo_CSG "Demo CSG")
 create_demo(Demo_DoF "Demo DoF")
 create_demo(Demo_VR "Demo Virtual Reality")
 create_demo(Demo_Texturing "Demo Texturing")
+create_demo(Demo_Marble "Demo Marble")

```

Comment:
I think we can just code to Demo texturing instead of creating many new Demo projects. Please check how Demo CSG, how the `buildScene***()` methods do the job

Reply:
I will take off the Demo Marble and build with the **buildScene()** way as suggested